master/buildbot/changes/gitpoller.py

```diff
@@ -159,8 +170,16 @@ def poll(self):
             '+%s:%s' % (self._removeHeads(branch), self._trackerBranch(branch))
             for branch in branches
         ]
-        yield self._dovccmd('fetch',
```

Comment:
I did a few grep searches and found a utility Mixin for this - logging.LoggingMixin.